onboarding-checklist.md

```diff
@@ -54,11 +54,14 @@ make sure you're assigned to this issue.
 
 #### Mandatory training ####
 
-- [ ] Complete [PALMS Cybersecurity Training](https://etms.hq.dhs.gov),
+- [ ] Complete ~~[PALMS Cybersecurity Training](https://etms.hq.dhs.gov)~~,
```

Comment:
PALMS is dead.  I think you can just remove this item entirely.

The new "hotness" is here: [CISA Course Management Site](https://usdhs.sharepoint.com/sites/CISA_OCLO_Intranet/CISACourses)

I think the specific training is here: [Cybersecurity Awareness Training ](https://usdhs.sharepoint.com/sites/CISA_OCLO_Intranet/CISACourses/SitePages/DHS-Mandatory-Training---CSAT-FY22.aspx)

_Note: as soon as you document these URLs they will be rotated to new random values._ 

Feel free to resolve this comment In absentia.

Reply:
Please see commit fe41c6e.